src/main/kotlin/com/cognifide/gradle/aem/deploy/responses/ListResponse.kt

```diff
@@ -1,4 +1,4 @@
-package com.cognifide.gradle.aem.deploy
+package com.cognifide.gradle.aem.deploy.responses
```

Comment:
Okay, changed. However, plurals in package names are IMO okay, as long as given package contains homogeneous concepts.